src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -97,12 +221,12 @@ def _check_fixed_for_custom_binary(testcase, job_type, testcase_file_path):
         command, symbolized_crash_stacktrace, unsymbolized_crash_stacktrace)
     testcase.last_tested_crash_stacktrace = data_handler.filter_stacktrace(
```

Comment:
nit: let's pass this field directly instead of through testcase please.

Reply:
Done.